ui/client/datasets/annotations/dataOUT.test.js

```diff
@@ -20,7 +20,7 @@ describe('formatAnnotationsOUT', () => {
         geo_type: 'latitude',
         coord_format: 'lonlat',
         primary: false,
-        resolve_to_gadm: false,
+        resolve_to_gadm: true,
```

Comment:
dataOUT.test.js checks for resolve_to_gadm=true for all geo fields, to validate that we'll always send that property to the backend.